controller/src/bin/xcvradm.rs

```diff
@@ -291,6 +351,22 @@ enum Cmd {
         /// a "1" is printed. Otherwise a "0" is printed.
         #[arg(long)]
         all: bool,
+
+        /// Print the output in a parseable format.
+        #[arg(long, short, conflicts_with_all = ["all", "with", "without"])]
+        parseable: bool,
+
+        /// Select the output fields to be displayed.
+        #[arg(long, short, requires = "parseable")]
+        output: Vec<StatusFields>,
+
+        /// Character used to separate output fields. (Default: :)
```

Comment:
Tiny nit, but I might "quote" the actual ":" to make it a bit more clear. Same with the other spots.